frontend/src/components/experimentRuns/ExperimentRunTableColumns.tsx

```diff
@@ -0,0 +1,191 @@
+        title: 'Start date',
+        dataIndex: ['experimentRun', 'start_date'],
+        sorter: true,
+        width: 120,
+        render: (_, { experimentRun }) => {
+            return <div>{experimentRun.start_date}</div>
+        }
+    },
+    [ExperimentRunColumnID.LAUNCH]: {
+        columnID: ExperimentRunColumnID.LAUNCH,
+        title: 'Launch',
+        dataIndex: ['experimentRun', 'launch'],
+        sorter: true,
+        width: 180,
+        render: (_, { experimentRun }) => {
+            return (<div>
+                <ExperimentRunLaunchCard experimentRun={experimentRun} experimentRunLaunch={launchesById[experimentRun.id]} />
+            </div>)
+        }
+    }
+
+})
+
+export enum ExperimentRunFilterID {
+    ID = ExperimentRunColumnID.ID,
+    NAME = ExperimentRunColumnID.NAME,
+    RUN_TYPE = ExperimentRunColumnID.RUN_TYPE,
+    INSTRUMENT = ExperimentRunColumnID.INSTRUMENT,
+    INSTRUMENT_TYPE = ExperimentRunColumnID.INSTRUMENT_TYPE,
+    CONTAINER_BARCODE = ExperimentRunColumnID.CONTAINER_BARCODE,
+    START_DATE = ExperimentRunColumnID.START_DATE,
+    LAUNCH = ExperimentRunColumnID.LAUNCH
+}
+
+export const EXPERIMENT_RUN_FILTER_DEFINITIONS: { [key in ExperimentRunColumnID]: FilterDescription } = {
+    [ExperimentRunColumnID.ID]: {
+        type: FILTER_TYPE.INPUT_OBJECT_ID,
+        key: UNDEFINED_FILTER_KEY,
+        label: "ExperimentRun ID",
+    },
+    [ExperimentRunColumnID.NAME]: {
+        type: FILTER_TYPE.INPUT,
+        key: UNDEFINED_FILTER_KEY,
+        label: "Name",
+    },
+    [ExperimentRunColumnID.RUN_TYPE]: {
+        type: FILTER_TYPE.INPUT,
+        key: UNDEFINED_FILTER_KEY,
+        label: "Run Type",
+    },
+    [ExperimentRunColumnID.INSTRUMENT]: {
+        type: FILTER_TYPE.INPUT,
+        key: UNDEFINED_FILTER_KEY,
+        label: "Instrument",
+    },
+    [ExperimentRunColumnID.INSTRUMENT_TYPE]: {
+        type: FILTER_TYPE.INPUT,
+        key: UNDEFINED_FILTER_KEY,
+        label: "Instrument Type",
+    },
+    [ExperimentRunColumnID.CONTAINER_BARCODE]: {
+        type: FILTER_TYPE.INPUT,
+        key: UNDEFINED_FILTER_KEY,
+        label: "Container Barcode",
+    },
+    [ExperimentRunColumnID.START_DATE]: {
+        type: FILTER_TYPE.INPUT,
+        key: UNDEFINED_FILTER_KEY,
+        label: "Start Date",
+    },
+    [ExperimentRunColumnID.LAUNCH]: {
+        type: FILTER_TYPE.INPUT,
+        key: UNDEFINED_FILTER_KEY,
+        label: "Launch",
+    }
+}
+
+export const EXPERIMENT_RUN_FILTER_KEYS: { [key in ExperimentRunColumnID]: string } = {
+    [ExperimentRunColumnID.ID]: 'id',
+    [ExperimentRunColumnID.NAME]: 'name',
+    [ExperimentRunColumnID.RUN_TYPE]: 'run_type',
+    [ExperimentRunColumnID.INSTRUMENT]: 'instrument',
+    [ExperimentRunColumnID.INSTRUMENT_TYPE]: 'instrument_type',
+    [ExperimentRunColumnID.CONTAINER_BARCODE]: 'container_barcode',
+    [ExperimentRunColumnID.START_DATE]: 'start_date',
+    [ExperimentRunColumnID.LAUNCH]: 'launch',
```

Comment:
"launch" does not exist in the backend... Maybe you can use "run_processing_launch_time".
Once the run_processing is launch it is the displayed value and it might be interesting to order them by that value as well as filter them by these values on a datepicker... maybe you can bend the date filter to use timestamps too or make a distinct filter for timestamps. The backend filter definition for that attribute is wrong (FK_FILTER) for some reason. This will have to be fixed if you want to go that way ...